### PR TITLE
JoinBuildFarm.md: Add info about secure route exposal

### PR DESCRIPTION
**Pull request by LorbusChris**
This change extends the doc with info on how to expose
registry/console/api/apps with Let's Encrypt TLS certs.

Might need some more polishing
/hold
/cc @rvanderp3

### CODE REVIEW

File: clusters/JoinBuildFarm.md [hunk 1]
```diff
@@ -53,6 +53,94 @@ After the [service accounts](./build-clusters/vsphere1/ci) are created by `apply
...
+For clusters running on AWS, see [this document](https://cert-manager.io/docs/configuration/acme/dns01/route53/) for how to create an IAM account for with access to the Route53 service.
+
+Create a secret containing the credentials on https://vault.ci.openshift.org, e.g.:
+- name: `arm-build-route53-access`
+- key: `AWS_ACCESS_KEY_ID`
+  value: `XXXXXXXXXXXXXXX`
+- key: `AWS_SECRET_ACCESS_KEY`
+  value: `XXXXXXXXXXXXXXX`
+
+Next ensure thus secret is either synced to, or created manually in the `cert-manager` namespace on the cluster.
```
Comment (rvanderp3) [inline clusters/JoinBuildFarm.md:77]: Minor typo on `thus`

**Review by rvanderp3 — COMMENTED** [review]
This lgtm

**Comment by LorbusChris** [comment]
@hongkailiu mind taking a look as well?

**Comment by LorbusChris** [comment]
One thing worth adding here is a short paragraph about how to use the vault, and sync secrets from there into the cluster (I have created a collection and secret in the Vault, but I haven't set up any secret syncing to the build farm yet - I might need a pointer for that part)

**Pull request closed without merging**